metascore/Interface.mo

```diff
@@ -24,19 +24,11 @@ module {
         getGames               : query ()                                          -> async [(MPublic.GamePrincipal, MPublic.Metadata)];
         getTop                 : query (n : Nat)                                   -> async [MAccount.Score];
         getGameScores          : query (MPublic.GamePrincipal, ?Nat, ?Nat)         -> async [MAccount.Score];
-        getDetailedGameScores  : query (MPublic.GamePrincipal, ?Nat, ?Nat)         -> async [MAccount.DetailedScore];
```

Comment:
Not having access to this query method will be a pain for the frontend. We'll need a way forward here.